app/src/main/java/de/danoeh/antennapod/adapter/FeedItemlistAdapter.java

```diff
@@ -62,6 +62,8 @@ public FeedItemlistAdapter(Context context,
 
         if(UserPreferences.getTheme() == R.style.Theme_AntennaPod_Dark) {
             playingBackGroundColor = ContextCompat.getColor(context, R.color.highlight_dark);
+        } else if(UserPreferences.getTheme() == R.style.Theme_AntennaPod_TrueBlack) {
+            playingBackGroundColor = ContextCompat.getColor(context, R.color.highlight_trueblack);
```

Comment:
It would be nice to have an `attr` in `styles.xml` for this, so we don't need the same `if` statement in multiple places

Reply:
This is a nice advice. Do you mean `R.color.highlight_dark` would have different value for different themes? 
If so, `UserPreferences.getTheme() == R.style.Theme_AntennaPod_Dark` has to contain the case for `UserPreferences.getTheme() == R.style.Theme_AntennaPod_TrueBlack`.
In this case, maybe it's better to add a switch that turns "black" into "true black" instead of adding a new theme...

It would be kind if you would give some advice on this.

Reply:
You can add

```
<attr name="playing_background" format="color"/>
```
to `attrs.xml` and for each theme:

```
<item name="playing_background">@color/highlight_trueblack</item>
```
to `styles.xml`.

In Java, you can then use the same code for all themes (so there does not need to be an if). Maybe the background can also be added to the layout itself instead of the Java code. 

```

accepted
This should do the job:

TypedValue typedValue = new TypedValue();
Theme theme = context.getTheme();
theme.resolveAttribute(R.attr.playing_background, typedValue, true);
@ColorInt int playingBackGroundColor = typedValue.data;
```